web/components/feed/feed-comments.tsx

```diff
@@ -194,7 +191,6 @@ export function FeedComment(props: {
               of{' '}
               <OutcomeLabel
                 outcome={betOutcome ? betOutcome : ''}
-                value={(matchedBet as any).value}
```

Comment:
I removed this since it only matters for old numeric contracts and that's not worth putting extra denormalized fields in our database for.